docs/index.md

```diff
@@ -0,0 +1,18 @@
+
+# Welcome to aws_resource_validator
```

Comment:
what's this file?
it has the same text as in the `README`

Reply:
it's for GitHub Pages